src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/emails/CourtEmailFactory.kt

```diff
@@ -110,6 +111,30 @@ object CourtEmailFactory {
     }
 
     return when (action) {
+      BookingAction.CREATE -> {
+        if (booking.createdByPrison) {
```

Comment:
I can kinda see what you are doing just here just not convinced it is the right place.

The factory isn't really meant to have too much logic in it really e.g. if the court is only supposed to get this email when a booking is created by a prison user then only call the factory function when it is a prison user.